qiskit_nature/second_q/algorithms/excited_states_solvers/excited_states_eigensolver.py

```diff
@@ -34,7 +34,7 @@ class ExcitedStatesEigensolver(ExcitedStatesSolver):
 
     def __init__(
         self,
-        qubit_converter: QubitConverter,
+        qubit_converter: Union[QubitConverter, QubitMapper],
```

Comment:
```suggestion
        qubit_converter: QubitConverter | QubitMapper,
```
We should try to do it in this way consistently. I'll only mention it here once as a stand-in

Reply:
I think it was the only appearance of the old syntax, my mistake